templates/user/profile.html

```diff
@@ -2,7 +2,7 @@
 {% extends 'layout.html' %}
 
 {% block title %}{{ user|capitalize }}'s profile{% endblock %}
-{% block description %}{{ user|capitalize }} is on Neomad with {{ articles.count() }} articles published and travels tracking {% if user.allow_community %}enabled{% else %}disabled{% endif %}.{% endblock %}
+{% block description %}{{ user|capitalize }} is on Neomad with {{ articles.count() }} articles published and {% if user.allow_community %}sharing with the community{% else %}in hight privacy{% endif %}.{% endblock %}
```

Comment:
you mean _with high privacy_, but we could actually say _a private profile_ instead

Reply:
as you made this comment on your own code I'm not sure about what I have to do with this 😛